composer.json

```diff
@@ -33,8 +33,8 @@
         "fidry/cpu-core-counter": "^0.4.0",
         "netresearch/jsonmapper": "^1.0 || ^2.0 || ^3.0 || ^4.0",
         "nikic/php-parser": "^4.13",
-        "sebastian/diff": "^4.0",
-        "spatie/array-to-xml": "^2.17.0",
+        "sebastian/diff": "^4.0 || ^5.0",
+        "spatie/array-to-xml": "^2.17.0 || ^5.0",
```

Comment:
Looks like a typo, `spatie/array-to-xml` doesn't have the 5.0 release yet.

Reply:
Ops, my mistake, the merge/rebase doubled the edit on the subsequent line too.